docs/comparisons.md

```diff
@@ -0,0 +1,58 @@
+|:--|:--|:--|
+| Fabrikate can manages both static Kubernetes manifests or Helm charts by wrapping them into [Components](https://github.com/microsoft/fabrikate/blob/develop/docs/component.md). Helm charts in a component can have their values set via config .yaml or .json values that fabrikate will then generate static manifests. <br><br> A single fabrikate component can be mapped to multiple config files so that different sets of releases can be created and managed. <br><br> Multiple components can be combined into a [High Level Definitions](https://github.com/microsoft/bedrock/blob/master/docs/gitops-pipeline.md#deep-dive-high-level-definitions) for ease of management and repeatability. | The Helm Operator makes use of Helm Release CRDs to manage releases and overwrite default values of a helm chart. Multiple Helm Releases can point to the same Helm Chart and parameterize the values for each release. | N/A |
+
+### Deployment Requirements and Components
+Artifacts and dependencies to utilize each tool.
+| [Fabrikate](https://github.com/microsoft/fabrikate) | [Helm Operator](https://github.com/fluxcd/helm-operator) | [Kustomize](https://github.com/kubernetes-sigs/kustomize) |
+|:--|:--|:--|
+| A repository for Helm Charts, a repository for Components/High Level Definitions, a repository for generated manifests. <br><br> Optionally these could all be the same repository, but this is not recommended. <br><br> Additionally a CI/CD pipeline should be used with fabrikate to generate the final manifests. | A repository for Helm Charts and another repository for Helm Releases. <br><br> Optionally, these could be the same repository. | N/A |
+
+### Deployment Workflow
+An example workflow for deploying the latest version application to a cluster.
+| [Fabrikate](https://github.com/microsoft/fabrikate) | [Helm Operator](https://github.com/fluxcd/helm-operator) | [Kustomize](https://github.com/kubernetes-sigs/kustomize) |
+|:--|:--|:--|
+| Once an application image is built and published to a container repository, then the associated config values in the Component/HLD repository needs to have the image.tag value updated to the latest version, either manually or as an automated step after publishing the image in a CI/CD pipeline. Once the version is set, then the fabrikate generation pipeline should be triggered to generate the latest manifests with the updated image tag. Once the final manifests are generated, the GitOps operator should sync the changes onto the cluster. | Once an application image is built and published to a container repository, then the associated Helm Release needs to have the image.tag value updated to the latest version, either manually or as an automated step after publishing the image in a CI/CD pipeline. Once the Helm Release has the latest values, then the Helm Operator will deploy the latest image to the repository. | N/A |
```

Comment:
"then the associated config values in the component/hld repository needs to have the image.tag value updated..." -> the config value `image.tag` in the Component/HLD repository needs to be updated with the latest version, either manually....